lib/migrator.js

```diff
@@ -0,0 +1,34 @@
+var fse = require("fs-extra");
+var path = require('path');
+var glob = require("glob");
+
+function moveFile(sourceFilePath, targetFilePath) {
+  let targetFileDirectory = path.dirname(targetFilePath);
+  if (!fse.existsSync(targetFileDirectory)) {
+    console.info(`📁 Creating ${targetFileDirectory}`);
+    fse.mkdirSync(targetFileDirectory, { recursive: true })
+  }
+
+  console.info(`👍 Moving ${sourceFilePath} -> ${targetFilePath}`);
+  fse.renameSync(sourceFilePath, targetFilePath);
+}
+
+module.exports = class Migrator {
+  constructor(options) {
+    this.options = options;
+  }
+
+  async execute() {
+    let sourceComponentTemplatesPath = path.join(this.options.projectRoot, 'app/templates/components');
+    var sourceTemplateFilePaths = glob.sync(`${sourceComponentTemplatesPath}/**/*.hbs`);
+
+    sourceTemplateFilePaths.forEach(sourceTemplateFilePath => {
+      let sourceTemplatePathInApp = sourceTemplateFilePath.slice(this.options.projectRoot.length); // '/app/templates/components/nested1/nested-component.hbs'
```

Comment:
It's been ages since I've used Node on Windows but is there any considerations there with path names or should this just work?

Reply:
I'm not sure if this will cause problems on windows. If it does, we can improve in future PRs I think